playerbot/strategy/paladin/HolyPaladinStrategy.cpp

```diff
@@ -465,6 +469,10 @@ void HolyPaladinStrategy::InitCombatTriggers(std::list<TriggerNode*>& triggers)
     triggers.push_back(new TriggerNode(
         "enemy is close",
         NextAction::array(0, new NextAction("judgement", ACTION_NORMAL), NULL)));
+
+    triggers.push_back(new TriggerNode(
+        "blessing of sacrifice",
+        NextAction::array(0, new NextAction("blessing of sacrifice", ACTION_MEDIUM_HEAL + 8), NULL)));
```

Comment:
does this make sense to be used when not in combat?

Reply:
Code is in HolyPaladinStrategy::InitCombatTriggers